requirements.txt

```diff
@@ -64,4 +79,4 @@ virtualenvwrapper==4.7.2
 wcwidth==0.1.7
 webencodings==0.5.1
 zmq==0.0.0
-sendgrid-django
+
```

Comment:
Why have you removed  `sendgrid-django`,  `psycopg2==2.6.1` and `coveralls==1.2.0` from requirements ?? @borenho @rosemaina